src/uu/numfmt/src/numfmt.rs

```diff
@@ -279,83 +303,57 @@ fn handle_stdin(options: NumfmtOptions) -> Result<()> {
 }
 
 pub fn uumain(args: impl uucore::Args) -> i32 {
-    let args = args.collect_str();
-
-    let mut opts = Options::new();
-
-    opts.optflag("h", "help", "display this help and exit");
-    opts.optflag("V", "version", "output version information and exit");
-    opts.optopt(
-        "",
-        "from",
-        "auto-scale input numbers to UNITs; default is 'none'; see UNIT above",
-        "UNIT",
-    );
-    opts.optopt(
-        "",
-        "to",
-        "auto-scale output numbers to UNITs; see Unit above",
-        "UNIT",
-    );
-    opts.optopt(
-        "",
-        "padding",
-        "pad the output to N characters; positive N will right-align; negative N will left-align; padding is ignored if the output is wider than N",
-        "N"
-    );
-    opts.optflagopt(
-        "",
-        "header",
-        "print (without converting) the first N header lines; N defaults to 1 if not specified",
-        "N",
-    );
-
-    let matches = opts.parse(&args[1..]).unwrap();
-    if matches.opt_present("help") {
-        println!("{} {}", NAME, VERSION);
-        println!();
-        println!("Usage:");
-        println!("  {0} [STRING]... [OPTION]...", NAME);
-        println!();
-        print!(
-            "{}",
-            opts.usage("Convert numbers from/to human-readable strings")
-        );
-        println!(
-            "UNIT options:
-   none   no auto-scaling is done; suffixes will trigger an error
-
-   auto   accept optional single/two letter suffix:
-
-          1K = 1000, 1Ki = 1024, 1M = 1000000, 1Mi = 1048576,
-
-   si     accept optional single letter suffix:
-
-          1K = 1000, 1M = 1000000, ...
-
-   iec    accept optional single letter suffix:
-
-          1K = 1024, 1M = 1048576, ...
-
-   iec-i  accept optional two-letter suffix:
-
-          1Ki = 1024, 1Mi = 1048576, ..."
-        );
-
-        return 0;
-    }
-    if matches.opt_present("version") {
-        println!("{} {}", NAME, VERSION);
-        return 0;
-    }
+    let usage = get_usage();
+
+    let matches = App::new(executable!())
+        .version(VERSION)
+        .about(ABOUT)
+        .usage(&usage[..])
+        .after_help(LONG_HELP)
+        .arg(
+            Arg::with_name("from")
```

Comment:
For consistency, could you follow what is done elsewhere with
OPT_FROM 
and sort by alphabetical order? Example:
https://github.com/uutils/coreutils/blob/master/src/uu/mktemp/src/mktemp.rs#L30-L35
Thanks

Reply:
Or better, do like https://github.com/uutils/coreutils/pull/1712/files
(mod options { })

Reply:
Updated using the `mod options` style.

Reply:
@FelipeLema how do you feel? 
(sorry I said conflicting things, I am still a bit unsure :)

Reply:
ok, so this is about the following story (my personal view)

Say I'm reading some code one day and I stand on something like `option::extra_args::cleanup`.

I can infer certain things just when looking at this:
- `cleanup` is a (command line) option
- it's one of several
- they have some kind of (hierarchical) order, so I can narrow my scope if I'm dealing with this.

Now, if I want to look into related code, there's a couple of things I can do

When doing "go to definition" using my IDE (with either rust-analyzer or some ad-hoc tagging system) I can jump to both the definition _and the other related options_ because they're all in the same (sub-)module.

Why can't I do this with `OPT_`? Because relationship between options is either implicit or has to be _semantically_ understood. Either way, it's an overload that the dev has to pay.

Yes, packing options into modules make an almost insignificant difference each, but people (me) will thank you for it because I *know* that extra effort sums up when dealing with big code bases.

TL;DR: Classifying options into modules improves maintainability according to my experience. Specially in large code bases.

Reply:
(I also think that packing options in modules can be done in separate features / commits ... so go for it!)

Reply:
agreed, many thanks for your arguments :)

Reply:
@FelipeLema thank you for explaining your thoughtful reasoning